setup.cfg

```diff
@@ -16,4 +16,18 @@ profile = black
 combine_as_imports = True
 
 [tool:pytest]
-addopts = --cov-report= --cov=starlette --cov=tests -rxXs
+addopts =
+  --cov-report=term-missing:skip-covered
+  --cov=starlette
+  --cov=tests
+  -rxXs
+  --strict-config
+  --strict-markers
+  --cov-config tests/.ignore_lifespan
+xfail_strict=True
+filterwarnings=
+    error
+    ignore: "@coroutine" decorator is deprecated.*:DeprecationWarning
+    ignore: Using or importing the ABCs from 'collections' instead of from 'collections\.abc' is deprecated.*:DeprecationWarning
```

Comment:
```suggestion
    # https://github.com/graphql-python/graphene/issues/1055
    ignore: Using or importing the ABCs from 'collections' instead of from 'collections\.abc' is deprecated.*:DeprecationWarning
```